docs/fsharp/tutorials/getting-started/getting-started-visual-studio.md

```diff
@@ -5,168 +5,147 @@ keywords: visual f#, f#, functional programming
 author: dend
 manager: danielfe
 ms.date: 05/16/2016
-ms.topic: language-reference
+ms.topic: article
 ms.prod: visual-studio-dev14
 ms.technology: devlang-fsharp
 ms.assetid: 8db75596-19a9-4eda-b20d-a12d517c8cc1 
 ---
 
 # Getting Started with F# in Visual Studio
 
-> [!NOTE]
-This article is not up to date with the latest Visual Studio.  It will be updated.
+F# and the Visual F# tooling are supported in the Visual Studio IDE.  To begin, you should [download Visual Studio](https://visualstudio.com/downloads/).  The rest of the article will assume Visual Studio 2015 Community Edition.
 
-Visual Studio 2010 includes a new programming language, F#. F# is a multiparadigm language that supports functional programming in addition to traditional object-oriented programming and .NET concepts. The following examples introduce some of its features and syntax. The examples show how to declare simple variables, to write and test functions, to create tuples and lists, and to define and use a class.
+## Installing the Visual F# Tools
 
-Note: General Settings
+After Visual Studio is finished initializing, select **Custom** as shown here:
 
+![](../../media/vs2015-install-1.png)
 
-### To create a new console application
+Select the Visual F# Tools under Programming Languages here:
 
-1. On the **File** menu, point to **New**, and then click **Project**.
-<br />
-
-2. If you cannot see Visual F# in the **Templates Categories** pane, click **Other Languages**, and then click **Visual F#**. The **Templates** pane in the center lists the F# templates.
-<br />
+![](../../media/vs2015-install-2.png)
 
-3. Look at the top of the **Templates** pane to make sure that **.NET Framework 4** appears in the **Target Framework** box.
-<br />
+Feel free to customize your installation further, and then continue with the installation.  After a while, Visual Studio will complete installation and you can create an F# project!
 
-4. Click **F# Application** in the list of templates.
-<br />
+## Creating a Console Application
 
-5. Type a name for your project in the **Name** field.
-<br />
+One of the most basic projects in Visual Studio is the Console Application.  Here's how to do it.  Once Visual Studio is open:
 
-6. Click **OK**.
-<br />  The new project appears in **Solution Explorer**.
-<br />
+1. On the **File** menu, point to **New**, and then click **Project**.
```

Comment:
For accessibility requirements, instead of "point to" and "click" we should use "choose".